src/sagas.js

```diff
@@ -323,7 +335,7 @@ function* fetchVmsByCountVLower (action) {
     if (!shallowFetch) {
       yield fetchConsoleMetadatas({ vms: internalVms })
       yield fetchDisks({ vms: internalVms })
-      yield fetchVmsCDRom({ vms: internalVms })
+      yield fetchVmsCdRom({ vms: internalVms, current: false })
```

Comment:
Likewise.

Reply:
This is correct, `current: false` should be passed.  `fetchVmsCdRom` was wrong.